docs/staticAnalysis.md

```diff
@@ -120,9 +120,9 @@ To set it up, [install node.js](https://nodejs.org/en/download/) if necessary (v
 ```
 ./gradlew installEslint
 
-# Alternatively, if you want to install the ESLint module globally, use the install command manually
-# Remember to use the correct tool version
-npm install -g eslint@{version}
+# Alternatively, if you want to install the ESLint module globally, use the install command manually to install both eslint and eslint-plugin-json
```

Comment:
OK if that feels confusing. Deleted that part and I guess the install instruction itself is self-explanatory